lib/widgets/vault_status_banner.dart

```diff
@@ -115,6 +120,24 @@ class VaultStatusBanner extends ConsumerWidget {
     );
   }
 
+  /// Build status banner for owner-steward state
+  /// This is when the owner has deleted their content but kept their shard
+  Widget _buildOwnerStewardStatus(BuildContext context, Vault vault) {
```

Comment:
I don't think I want a special status for this. I think we should use the existing "ready for recovery" state but maybe we need to modify the message to say that you or a steward can initiate recovery at any time.